src/app/screens/Calculator/layout.js

```diff
@@ -1,21 +1,29 @@
 import React from 'react';
 import { string, arrayOf, elementType } from 'prop-types';
+import { push } from 'connected-react-router';
+import { useDispatch } from 'react-redux';
 
 import styles from './styles.module.scss';
 
-const CalculatorLayout = ({ result, digits, calc, operators }) => (
-  <div>
-    <div className={styles.calculatorDisplay}>
-      {result ? <div className={styles.result}>{result}</div> : ''}
-      <div>{calc || 0}</div>
-    </div>
-    <div className={styles.buttons}>
-      <div className={styles.digits}> {digits}</div>
-      <div className={styles.operators}>{operators}</div>
-    </div>
-  </div>
-);
+const CalculatorLayout = ({ result, digits, calc, operators }) => {
+  const dispatch = useDispatch();
 
+  return (
+    <div>
+      <div className={styles.calculatorDisplay}>
+        {result ? <div className={styles.result}>{result}</div> : ''}
+        <div>{calc || 0}</div>
+      </div>
+      <div className={styles.buttons}>
+        <div className={styles.digits}> {digits}</div>
+        <div className={styles.operators}>{operators}</div>
+      </div>
+      <button className={styles.calcToRecButton} onClick={() => dispatch(push('/record'))}>
```

Comment:
instead of doing `dispatch(push('/record'))` import RECORD from constants/routes and use the already declared constant. Then you'll have somthing like this:
dispatch(push(RECORD))